sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/IntervalExpressionsSuite.scala

```diff
@@ -197,10 +198,22 @@ class IntervalExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
   }
 
   test("multiply") {
-    def check(interval: String, num: Double, expected: String): Unit = {
-      checkEvaluation(
-        MultiplyInterval(Literal(stringToInterval(interval)), Literal(num)),
-        if (expected == null) null else stringToInterval(expected))
+    def check(
+        interval: String,
+        num: Double,
+        expected: String,
+        checkException: Boolean = false): Unit = {
+      Seq("true", "false").foreach { v =>
+        withSQLConf(SQLConf.ANSI_ENABLED.key -> v) {
+          if (checkException) {
```

Comment:
We don't need the `checkException` flag
```
if (expected == null) {
  checkExceptionInExpression
} else {
  ...
}
```

Reply:
BTW we can add a `val expr = MultiplyInterval(Literal(stringToInterval(interval)), Literal(num))` at the beginning, to avoid code duplication.